python/scripts/load_data.py

```diff
@@ -1,227 +0,0 @@
-"""
```

Comment:
We should have something in the /scripts file that we can use to rebuild the database - essentially the same code you have after the if name=main statement in the /ingestion/load_data.py file.  That way we can keep the similar workflow where the /scripts folder is where people go to find executables if they don't need to know how the housinginsights package works.